scripts/milestone.js

```diff
@@ -0,0 +1,70 @@
+// npm install github-api
+let Issue = require('../node_modules/github-api/dist/components/Issue.js')
+let config = require('../config.json')
+
+// create milestones all
+let createMilestoneAll = (title, myDate) => {
+  config.repos.forEach(repo => {
+    let issueObj = new Issue(repo.owner+'/'+repo.repo, {token: process.env.GITHUB_TOKEN})
+    let milestoneData = {
+      "title": "Sprint - "+title,
+      "state": "open",
+      "description": "",
+      "due_on": myDate
+    }
+    return issueObj.createMilestone(milestoneData)
+  })
+}
+// create milestone 
+let createMilestone = (owner, repo, title, myDate) => {
```

Comment:
I would merge `createMilestoneAll` and `createMilestone` and have simple if statement there - Eg:
```
let createMilestone = (title, myDate, owner, repo) => {
   if (!owner || !repo) {
   config.repos.forEach(repo => {
    let issueObj = new Issue(repo.owner+'/'+repo.repo, {token: process.env.GITHUB_TOKEN})
    let milestoneData = {
      "title": "Sprint - "+title,
      "state": "open",
      "description": "",
      "due_on": myDate
    }
    return issueObj.createMilestone(milestoneData)
  })
  } else {
   create single....
  }
}
```

Reply:
@zelima  wonderful suggestion, refactored, worked fine.